package.json

```diff
@@ -2,11 +2,14 @@
   "name": "weasel.js",
   "version": "0.0.1",
   "description": "Collection of UI elements that behave as popovers",
-  "main": "index.js",
+  "main": "dist/index",
+  "types": "dist/index",
   "scripts": {
+    "build": "tsc",
     "test": "mocha 'test/browser/**/*.{js,ts}'",
     "test-debug": "mocha --bail test/browser/**/*.{js,ts} --no-exit",
-    "test-manual": "webpack-dev-server --config test/fixtures/webpack.config.js"
+    "test-manual": "webpack-dev-server --config test/fixtures/webpack.config.js",
+    "prepare": "npm run build"
```

Comment:
You may have gotten it from grainjs, but I think that one is funky because we are installing it in Grist from a github URL. The example of grain-rpc or ts-interface-checker might be better here. In either case, the key element is the "files" key, which should include "dist" (but I'd say include lib/ as well as grainjs does).

Reading more on this, I'd also suggest replacing this "prepare" line with "prepublish". This way running "npm publish" would automatically run the build (in grain-rpc and ts-interface-builder I think I just have to remember to do it before running npm publish).